### ISSUE TITLE
endless read rpm file via rpmfile.open()

### ISSUE THREAD

**Issue by artur-barsegyan** [srossross/rpmfile#18]
Via strace i got this:
read(3, ..., 4096) = 0

**Comment by agronick** [comment]
This
```
        if char is None or char == '':
            raise RPMError("reached end of file without finding magic char \x8e")
```
should be
```
        if char is None or char == b'':
            raise RPMError("reached end of file without finding magic char \x8e")
```
With the string prefixed with b.

**Comment by pdxjohnny** [comment]
Closed via 777de8ba8349a8c54ece42b4778eb4ec70999734
